### PR TITLE
Add initial CLA document

### PR DESCRIPTION
**Pull request by aalmiray**
Initial CLA definition.

Once agreed upon, all PRs may be checked for signed CLAs from contributors by registering at https://cla-assistant.io/
Only repo admins may perform this action.

### CODE REVIEW

**Comment by ebullient** [comment]
Clarification because I know next to nothing process-wise:

- https://cla-assistant.io/ suggests a gist. 
  - Does it matter where the CLA lives? 
  - Can other repositories (not in the commonhaus organization) reference this text?

- Assuming we have other folks in favor, we should update #78 to suggest Commonhaus CLA (instead of any-old CLA) so that CF projects are using the same language.

**Comment by aalmiray** [comment]
Ah yes, I now remember I have a copy of the JReleaser CLA as a gist so that CLAassistant can see it.

I believe the `CLA.md` could be placed anywhere within the CF repositories.
As it is currently worded this CLA applies only to CF itself. It wouldn't make sense for other projects to link to it.
If this text is generalized to any project (within the CF [not necessarily within the CF GitHub org]) then I suppose it should be OK for projects to link to it.

**Review by kenfinnigan — APPROVED** [review]
As a CLA for CF directed contributions, looks good

**Comment by ebullient** [comment]
Ok. I have a thought. Can you move this within the policies folder?
I think I can use a build to sync it with a gist..

**Comment by ebullient** [comment]
DONE: 

- https://gist.github.com/commonhaus-bot/5fe1c9c28e0899d9a272c900bbe89651
- https://github.com/commonhaus/foundation-draft/blob/main/.github/workflows/cla.yaml

File: policies/CLA.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+## Definitions
+
+1. "You" (or "Your") shall mean the copyright owner or legal entity authorized by the copyright owner that is making this
+Agreement with Project. For legal entities, the entity making a Contribution and all other entities that control, are
+controlled by, or are under common control with that entity are considered to be a single Contributor. For the purposes
+of this definition, "control" means (i) the power, direct or indirect, to cause the direction or management of such entity,
+whether by contract or otherwise, or (ii) ownership of fifty percent (50%) or more of the outstanding shares, or
+(iii) beneficial ownership of such entity.
+"Contribution" shall mean any original work of authorship, including any modifications or additions to an existing work,
```
Comment (trustin) [inline policies/CLA.md:20]: We could insert an empty line between these two lines.

Comment (ebullient) [reply policies/CLA.md:20]: Make a PR? Happy to take that. I merged (ahead of lawyer review, etc.) so I could get the tooling working.

Comment (aalmiray) [reply policies/CLA.md:20]: There was an empty line and the markdown linter complained 🤷

Comment (ebullient) [reply policies/CLA.md:20]: Should only complain about dangling white space at the end of the line.. or two blank lines in a row. Will check and fix.

**Pull request merged**